airtime_mvc/application/controllers/PreferenceController.php

```diff
@@ -262,6 +262,41 @@ public function streamSettingAction()
                     //Application_Model_RabbitMq::PushSchedule();
                 }
 
+                // pulling this from the 2.5.x branch
+                if (!Application_Model_Preference::GetMasterDjConnectionUrlOverride()) {
+                    $master_connection_url = "http://".$_SERVER['SERVER_NAME'].":".$values["master_source_port"].$values["master_source_mount"];
+                    if (empty($values["master_source_port"]) || empty($values["master_source_port"])) {
```

Comment:
This should probably be:

```php
if (empty($values["master_source_port"]) || empty($values["master_source_mount"])) {
```

I kinda liked how SaaS was using `parse_url(`) on `Application_Model_Preference::GetMasterDJSourceConnectionURL()` to achieve this without only keeping `MasterDJSourceConnectionURL` for display purposes.